intellij/test/junit/saros/intellij/filesystem/IntelliJReferencePointManagerTest.java

```diff
@@ -0,0 +1,107 @@
+package saros.intellij.filesystem;
+
+import com.intellij.openapi.module.Module;
+import com.intellij.openapi.roots.ModuleRootManager;
+import com.intellij.openapi.vfs.VirtualFile;
+import org.easymock.EasyMock;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import saros.filesystem.IPath;
+import saros.filesystem.IReferencePoint;
+
+public class IntelliJReferencePointManagerTest {
+
+  IReferencePoint referencePoint;
+  IPath fileReferencePointRelativePath;
+  IPath folderReferencePointRelativePath;
+  IPath projectFullPath;
+  IntelliJReferencePointManager intelliJReferencePointManager;
+  VirtualFile file;
+  VirtualFile folder;
+  VirtualFile moduleRoot;
+  String moduleRootPath;
+  ModuleRootManager manager;
+
+  @Before
+  public void setup() {
+    // moduleRootPath = "path/to/foo";
+    moduleRootPath = "";
+    referencePoint = EasyMock.createMock(IReferencePoint.class);
```

Comment:
You override the value of `referencePoint` in other tests. Rather introduce a private method that is called if you need this value (would make line 15 obsolete).

Reply:
I rewrote the tests